integration_test/install_test.go

```diff
@@ -105,6 +105,17 @@ func TestKrewInstall_Manifest(t *testing.T) {
 	test.AssertExecutableInPATH("kubectl-" + validPlugin)
 }
 
+func TestKrewInstall_ManifestURL(t *testing.T) {
```

Comment:
Maybe also add a test-case where `--manifest-url` _and_ `--archive` are given? What should happen in that case?

Reply:
I think we can make these flags mutually exclusive and explicitly error out. People most probably won't even use it.

That said, --archive just always uses the provided file instead of downloading the .zip file from the internet. So it's actually still valid here, but that's not why people are going to use --manifest-url for. 

So yeah, I recommend 1) explicitly fail in this case 2) add integration tests to test that.

Reply:
Done